package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "systers.github.io",
+  "version": "1.0.0",
+  "description": "[![Travis](https://img.shields.io/travis/rust-lang/rust.svg?style=plastic)](https://github.com/systers/systers.github.io)",
+  "main": "index.js",
+  "scripts": {
+    "start": "ecstatic --port=8050 .",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/systers/systers.github.io.git"
+  },
+  "author": "",
+  "license": "UNLICENSED",
```

Comment:
@Janiceilene I have a doubt here, in our project the LICENSE file points to  GNU GPL License, https://github.com/systers/systers.github.io/blob/master/LICENSE
So are we good with using the `UNLICENSED` in the node module?

Reply:
The GNU GPL license isn't necessarily correct.  I need to talk with @MeepyMay and figure out what license should be used for the project, but she's out of town and I wanted to push this asap, so I went with unlicensed for now.

Reply:
Yeah, we didn't decide. Can you update on the next PR? I don't think we'll have the bandwidth to research all of the options. So we can add the GPU 2.0 for all repos, except the crowdmap/mailman3-related repos. They have MIT's license.